modules/check_in/app/serializers/check_in/v2/appointment_data_serializer.rb

```diff
@@ -8,6 +8,7 @@ class AppointmentDataSerializer
       set_id(&:id)
       set_type :appointment_data
 
+      # rubocop:disable Metrics/BlockLength
```

Comment:
The metric will be turned back on after a refactor of this code in the near future.